hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/common/model/HoodieSparkRecord.java

```diff
@@ -89,6 +89,8 @@ public class HoodieSparkRecord extends HoodieRecord<InternalRow> {
    */
   private final transient StructType schema;
 
+  private Comparable<?> orderingValue;
```

Comment:
should we make it `transient` here, not sure Kyro serialization will ignore this.

Reply:
to confirm: Kyro would ignore the `transient` value